backend/pyproject.toml

```diff
@@ -82,7 +80,11 @@ pytest-xdist = "3.6.1"
 [tool.poetry.group.lint.dependencies]
 # Dependencies only required for linting.
 pre-commit = "4.2.0"
-ruff = "0.11.5"
+ruff = "^0.11.9"
+
+
+[tool.poetry.group.dev.dependencies]
+pypdf2 = "^3.0.1"
```

Comment:
Why is this added?

Reply:
Should also not be in the "dev" group, use more specific groups

Reply:
Can not say, added by accident? 😅